Packs/CrisisManagement/pack_metadata.json

```diff
@@ -1,21 +1,26 @@
 {
-  "name": "Crisis Management",
-  "description": "Manage crises such as pandemics or other events that would change the conditions under which employees would normally work. Note: This is a beta pack, which lets you implement and test pre-release software. Since the pack is beta, it might contain bugs. Updates to the pack during the beta phase might include non-backward compatible features. We appreciate your feedback on the quality and usability of the pack to help us identify issues, fix them, and continually improve.",
-  "support": "xsoar",
-  "currentVersion": "1.0.0",
-  "author": "Cortex XSOAR",
-  "url": "https://www.paloaltonetworks.com/cortex",
-  "email": "",
-  "created": "2020-05-06T08:54:47Z",
-  "categories": [
-    "Case Management"
-  ],
-  "tags": [
-    "Crisis",
-    "Pandemic",
-    "Remote Work",
-    "COVID-19"
-  ],
-  "useCases": [],
-  "keywords": []
+    "name": "Crisis Management",
+    "description": "Manage crises such as pandemics or other events that would change the conditions under which employees would normally work. Note: This is a beta pack, which lets you implement and test pre-release software. Since the pack is beta, it might contain bugs. Updates to the pack during the beta phase might include non-backward compatible features. We appreciate your feedback on the quality and usability of the pack to help us identify issues, fix them, and continually improve.",
+    "support": "xsoar",
+    "certification": "",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "categories": [
+        "Case Management"
+    ],
+    "tags": [
+        "Crisis",
+        "Pandemic",
+        "Remote Work",
+        "COVID-19"
+    ],
+    "created": "2020-05-06T08:54:47Z",
+    "updated": "2020-05-06T08:54:47Z",
+    "beta": true,
+    "deprecated": false,
+    "useCases": [],
+    "keywords": [],
+    "dependencies": {}

```

Comment:
I will explain more about it later but this will override automatically calculated dependencies. So please remove `dependencies` field.